crates/rust-analyzer/src/config.rs

```diff
@@ -220,6 +220,9 @@ config_data! {
         /// Controls file watching implementation.
         files_watcher: FilesWatcherDef = "\"client\"",
 
+        /// Exclude imports from find-all-references.
+        findAllRefs_excludeImports: bool = "false",
```

Comment:
```suggestion
        references_excludeImports: bool = "false",
```
I'd prefer this name for the config

Reply:
renamed